src/backend/api/meals.js

```diff
@@ -2,14 +2,56 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+router.get("/", async (req, res) => {
   try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const titles = await knex.select().table('meal')
+    res.json(titles);
   } catch (error) {
-    throw error;
+    throw res.status(404).json(error);
   }
 });
 
+router.post("/", async(req, res) => {
+  try {
+    const insertData = await knex('meal').insert({ title: 'Lunch', description: 'Full Lunch', location: 'vallenbæk', when: '2022-09-17', max_reservations: 5, price: 200, created_date: '2022-09-17' });
```

Comment:
You should take request body as input and not insert hardcoded values

Reply:
I have update code